setup.py

```diff
@@ -74,15 +74,15 @@
     'Faker>=2.0.3',
     'Flask-BabelEx>=0.9.4',
     'idutils>=1.1.3',
-    'invenio-formatter[badges]>=1.0.2,<1.1.0',
-    'invenio-jsonschemas>=1.0.1,<1.1.0',
-    'invenio-pidstore>=1.1.0,<1.3.0',
-    'invenio-records-rest>=1.6.4,<1.7.0',
-    'invenio-records>=1.3.0,<1.4.0',
-    'invenio-records-files>=1.2.1,<1.3.0',
+    'invenio-formatter[badges]>=1.0.3',
```

Comment:
```suggestion
    'invenio-formatter[badges]>=1.1.0a1',
```

Reply:
The tag in invenio still shows 1.0.3 https://github.com/inveniosoftware/invenio/blob/v3.4.0a1/setup.py#L36 Which is where I got the deps versions. Should I check master (wondering about compatibilities)?